feathr_project/feathr/anchor.py

```diff
@@ -34,7 +34,7 @@ def to_feature_config(self) -> str:
         tm = Template("""
             {{anchor_name}}: {
                 source: {{source.name}}
-                key: [{{key_list}}]
+                key.sqlExpr: [{{key_list}}]
```

Comment:
In general this makes sense, but is there any effect on compatibility? Is the change motivated by performance?

Reply:
No. It is compatible. Yes, It is motivated by performance.